src/commonMain/kotlin/com/github/quillraven/fleks/entity.kt

```diff
@@ -310,12 +322,16 @@ class DefaultEntityProvider(
     override operator fun minusAssign(entity: Entity) {
         recycledEntities.add(entity)
         removedEntities.set(entity.id)
+        currentEntityVersions[entity.id] = entity.version
```

Comment:
I think this line is not necessary because the version is only updated in the `create` method before. Assigning it here once more is redundant?